backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/blobs/CassandraBlobModule.java

```diff
@@ -0,0 +1,53 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.backends.cassandra.blobs;
+
+import java.util.List;
+
+import org.apache.james.backends.cassandra.components.CassandraModule;
+import org.apache.james.backends.cassandra.components.CassandraTable;
+import org.apache.james.backends.cassandra.components.CassandraType;
+
+import com.datastax.driver.core.DataType;
+import com.datastax.driver.core.schemabuilder.SchemaBuilder;
+import com.google.common.collect.ImmutableList;
+
+public class CassandraBlobModule implements CassandraModule{
+
+    public static final String TABLE_NAME = "blobs";
+    public static final String ID = "id";
+    public static final String PART = "part";
+    public static final String DATA = "data";
+
+    @Override
+    public List<CassandraTable> moduleTables() {
+        return ImmutableList.of(new CassandraTable(TABLE_NAME,
+            SchemaBuilder.createTable(TABLE_NAME)
+                .ifNotExists()
+                .addPartitionKey(ID, DataType.timeuuid())
+                .addClusteringColumn(PART, DataType.bigint())
```

Comment:
However we can also proceed with two tables:

```
Table (blobPartIds) {
 UUID blobId,
 UUID partId,
 PK(blobId, partId)
}

And

Table (blobParts) {
  UUID partId,
  Blob data
}
```

And use the exact same mechanism: load partIds for a blob first, then Stream like it was done here. (Maybe this new design is better).